Doc/library/typing.rst

```diff
@@ -1266,213 +1318,319 @@ These can be used as types in annotations using ``[]``, each having a unique syn
 
    .. versionadded:: 3.11
 
-Building generic types
-""""""""""""""""""""""
+Building generic types and type aliases
+"""""""""""""""""""""""""""""""""""""""
 
 These are not used in annotations. They are building blocks for creating generic types.
```

Comment:
Type aliases _are_ used in annotations

Reply:
So are typevars and generics :)

The point is that the names `TypeVar` and `TypeAliasType` are not used in annotations themselves. Maybe that's not important to mention here, though.